src/node/internal/internal_buffer.ts

```diff
@@ -2688,9 +2688,13 @@ export function transcode(
   if (normalizedToEncoding === undefined) {
     throw new ERR_UNKNOWN_ENCODING(toEncoding);
   }
-  return Buffer.from(
-    bufferUtil.transcode(source, normalizedFromEncoding, normalizedToEncoding)
+
+  const u8: Uint8Array = bufferUtil.transcode(
```

Comment:
Why do we need this?

```suggestion
  const u8 = bufferUtil.transcode(
```

Reply:
Mostly for clarity since `bufferUtil.transcode(...)` used to return an `ArrayBuffer`